### DISCUSSION TITLE
https://soundgrail.com/blog/essential-music-theory-tips-for-guitarists-from-beginner-to-pro

### DISCUSSION THREAD

**Discussion by giscus** [Distalk/giscus#201] [Giscus]
# https://soundgrail.com/blog/essential-music-theory-tips-for-guitarists-from-beginner-to-pro

Discover essential music theory tips for guitarists, from beginner fundamentals to advanced techniques, to elevate your playing to a professional level.

https://soundgrail.com/blog/essential-music-theory-tips-for-guitarists-from-beginner-to-pro

**Comment by Feodok07** [comment]
I remember feeling totally stuck in a rut with my playing last year, just mindlessly running through the same tired pentatonic shapes every single evening. To help break out of it, I started actively trying to transcribe solos by ear during my commute. It's funny because while I was looking for the best platforms to stream deep cuts for practice, I spent an entire morning reading [youtube music reviews](https://youtube-music.pissedconsumer.com/review.html) to see if their algorithm is actually better for finding obscure jazz tracks than other services.
